ros2bag/ros2bag/verb/record.py

```diff
@@ -74,6 +86,8 @@ def main(self, *, args):  # noqa: D102
 
         self.create_bag_directory(uri)
 
+        self.validate_splitting_size(args)
```

Comment:
I don't see the purpose of doing this within the ros2bag package. In the end, every storage should validate that splitting option itself and so I think it makes more sense to have this implemented on a per storage level.
Saying I don't see the point to do this only for sqlite3 and simply ignore all other plugins.